### DISCUSSION TITLE
What song did the president of 🇰🇷 sing during his recent visit to the White House?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#737] [Galaxy Trivia] [answered]
**Bonus:** Write the first verse

**Comment by peterbaccaro** [comment, marked as answer]
South Korea's president sang **Don McLean's “American Pie”** to great applause as he concluded a visit to the White House on Wednesday 26 April. Yoon Suk Yeol - who is known back in his home nation to be something of a karaoke buff – shared his love of American music during a state dinner with Joe Biden.

The first verse is:

**"A long, long time ago
I can still remember
How that music used to make me smile
And I knew if I had my chance
That I could make those people dance
And maybe they'd be happy for a while"**

**Comment by thuongtruong1009** [comment]
South Korea's president sings 'American Pie' during White House visit 
APRIL 26: South Korean President Yoon Suk-yeol sings "American Pie" by Don McLean alongside U.S. President Joe Biden during a state dinner at the White House, April 26, 2023 in Washington, DC. President Joe Biden and first lady Jill Biden are hosting South Korean President Yoon Suk-yeol and South Korean first lady Kim Keon-hee for a State Dinner. (Photo by Chip Somodevilla/Getty Images)

**Comment by SANKARAMDAS** [comment]
South Korean President Yoon Suk-yeol sings Don McLean's “American Pie” as President Joe Biden and the guests listen in the State Dining Room at the White House in Washington, DC. 
When the singers finished, Joe Biden and Yoon Suk Yeol took the stage.
